src/centralellile/_2017_2018/poo4/seance1/tests/Test1.java

```diff
@@ -1,68 +1,67 @@
 /*
  * The MIT License
- *
+ * To change this template file, choose Tools | Templates
```

Comment:
Ligne en trop je pense

Reply:
j'ai pris all language

Reply:
j'ai ceci dans mes fichiers comme licence à l'intérieur :

Reply:
/*
 * The MIT License
 * To change this template file, choose Tools | Templates
 * Copyright 2018 Team SI.
 * and open the template in the editor.
 *        
 * Permission is hereby granted, free of charge, to any person obtaining a copy       
 * of this software and associated documentation files (the "Software"), to deal      
 * in the Software without restriction, including without limitation the rights       
 * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell      
 * copies of the Software, and to permit persons to whom the Software is      
 * furnished to do so, subject to the following conditions:       
 *        
 * The above copyright notice and this permission notice shall be included in     
 * all copies or substantial portions of the Software.        
 *        
 * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR     
 * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,       
 * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE        
 * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER     
 * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,      
 * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN      
 * THE SOFTWARE.      
 */

Reply:
protip : tu peux mettre du code entre \`\`\` mon code  \`\`\` pour qu'il soit mieux lisible
```
example();
code.formated();
```

Reply:
````
The MIT License
 * To change this template file, choose Tools | Templates
 * Copyright 2018 Team SI.
 * and open the template in the editor.
```
y'a un problème 
` To change this template file, choose Tools | Templates and open the template in the editor.` c'est le template par défaut 
ça a été remplacé
[cf fichier branche upstream](https://github.com/centralelille/2017-2018-poo4-seance1/blob/master/src/centralellile/_2017_2018/poo4/seance1/modele/Service.java)

là t'a un mix des 2. Si tu lis les 3 lignes à la suite, tu peux constater que ça ne veut rien dire.

Reply:
Pour ceci :
protip : tu peux mettre du code entre ``` mon code ``` pour qu'il soit mieux lisible

example();
code.formated();


Je ne sais pas